Build Error on Ubuntu18.04 --- grpc-web version: 0.4.0

## Error Log:
g++ -std=c++11 -I/usr/local/include -pthread  -c -o grpc_generator.o grpc_generator.cc
grpc_generator.cc: In function 'std::__cxx11::string grpc::web::{anonymous}::GetRootPath(const string&, const string&)':
grpc_generator.cc:226:20: error: 'count' is not a member of 'std'
   size_t slashes = std::count(from_filename.begin(), from_filename.end(), '/');
                    ^~~
<builtin>: recipe for target 'grpc_generator.o' failed
make[1]: *** [grpc_generator.o] Error 1

## Command:
make install-plugin

It didn't work, still the same error. Do you need any env information else?

Actually, another problem occurs before this problem.

```shell
cd "/go/grpc-web-0.4.0"/javascript/net/grpc/web && make install
make[1]: Entering directory '/go/grpc-web-0.4.0/javascript/net/grpc/web'
g++ -std=c++11 -I/usr/local/include -pthread  -c -o grpc_generator.o grpc_generator.cc
grpc_generator.cc:19:53: fatal error: google/protobuf/compiler/code_generator.h: No such file or directory
 #include <google/protobuf/compiler/code_generator.h>
                                                     ^
compilation terminated.
<builtin>: recipe for target 'grpc_generator.o' failed
```

And then I run `apt-get install libprotoc-dev` fix this.

--- a/javascript/net/grpc/web/grpc_generator.cc
+++ b/javascript/net/grpc/web/grpc_generator.cc
@@ -16,6 +16,7 @@
  *
  */
 
+#include <algorithm>
 #include <google/protobuf/compiler/code_generator.h>
 #include <google/protobuf/compiler/plugin.h>
 #include <google/protobuf/descriptor.h>

